### PR TITLE
Fixed more typos and xmldoc issues

### PR DESCRIPTION
**Pull request by NathanBaulch**

### CODE REVIEW

File: src/Quartz/Impl/Calendar/DailyCalendar.cs [hunk 1]
```diff
@@ -105,6 +105,8 @@ public class DailyCalendar : BaseCalendar
         ///     </li>  
         /// </ul>
         /// </summary>
+        /// <param name="rangeStartingTime">The range starting time in millis.</param>
```
Comment (lahma) [inline src/Quartz/Impl/Calendar/DailyCalendar.cs]: if I remember right this will give compilation exception?

Comment (NathanBaulch) [inline src/Quartz/Impl/Calendar/DailyCalendar.cs]: i'm pretty sure this is the normal way to document parameters. `<param>` is used to describe parameters and `<paramref>` is used inside `<summary>` blocks to reference parameters by name.

Comment (lahma) [inline src/Quartz/Impl/Calendar/DailyCalendar.cs]: You don't get compile error after this?

Comment (NathanBaulch) [inline src/Quartz/Impl/Calendar/DailyCalendar.cs]: No, what error are you seeing?

**Comment by lahma** [comment]
My mistake, checked and compiles fine. Thanks for the contribution!

**Pull request merged**